notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>migrate to core? also "args" -&gt; "Args"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I already migrated it, but forgot to delete the original one. good catch!</p>